compiler/next/tools/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Copyright 2021 Hewlett Packard Enterprise Development LP
+# Copyright 2022 Hewlett Packard Enterprise Development LP
```

Comment:
@mppf - do you know why this file has the copyright in it twice?

Reply:
Nope I think it was an accident